unimol/unimol/utils/docking_utils.py

```diff
@@ -102,8 +94,51 @@ def load_lmdb_data(lmdb_path, key):
     return collects
 
 
-def docking_data_pre(raw_data_path, predict_path):
+def reprocess_content(content: Dict, base_mol: Optional[Chem.Mol] = None, M: int = 2000, N: int = 10, mmff: bool = False, seed: int = 42, stereo_from3d: bool = True) -> Dict:
+    """ Reprocess a data point in the LMDB schema for Docking usage. Ensures correct stereochemistry.
+    Basic principle is to perceive stereochem from label molecule's 3D and keep it intact.
+    Use default values for best results
+
+    Args:
+        content: A dictionary of the LMDB schema. (atoms, holo_mol, mol_list, cooredinates, etc.)
+        base_mol: The molecule to replace the holo_mol with, if passed
+        M: The number of conformers to generate
+        N: The number of clusters to group conformers and pick a representative from
+        mmff: Whether to use MMFF minimization after conformer generation
+        seed: The random seed to use for conformer generation
+        stereo_from3d: Whether to perceive stereochemistry from the 3D coordinates of the label molecule
 
+    Returns:
+        A copy of the original, with the holo_mol replaced with the base_mol, and coordinates added.
+    """
+    if base_mol is None:
+        base_mol = content["holo_mol"]
+    # Copy so we don't change inputs
+    content = copy.deepcopy(content)
+    base_mol = copy.deepcopy(base_mol)
+    base_mol = Chem.AddHs(base_mol, addCoords=True)
+    # assign stereochem from 3d
+    if stereo_from3d and base_mol.GetNumConformers() > 0:
```

Comment:
keeps the stereochemistry fixed if wanted -> removes tetrahedral chirality mismatches between labels and predictions